types/fh-sync.d.ts

```diff
@@ -0,0 +1,121 @@
+declare module 'fh-sync' {
+
+  export interface PendingWorkerBackoff {
+    strategy: string;
+    max: string;
+  }
+
+  export interface AckWorkerBackoff {
+    strategy: string;
+    max: string;
+  }
+
+  export interface SyncWorkerBackoff {
+    strategy: string;
+    max: number;
+  }
+
+  export interface SyncGlobalOptions {
+    pendingWorkerInterval: number;
+    pendingWorkerConcurrency: number;
+    pendingWorkerBackoff: PendingWorkerBackoff;
+    ackWorkerInterval: number;
+    ackWorkerConcurrency: number;
+    ackWorkerBackoff: AckWorkerBackoff;
+    syncWorkerInterval: number;
+    syncWorkerConcurrency: number;
+    syncWorkerBackoff: SyncWorkerBackoff;
+    schedulerInterval: number;
+    schedulerLockMaxTime: number;
+    schedulerLockName: string;
+    datasetClientUpdateConcurrency: number;
+    collectStats: boolean;
+    statsRecordsToKeep: number;
+    collectStatsInterval: number;
+    metricsInfluxdbHost?: string;
+    metricsInfluxdbPort?: number;
+    metricsReportConcurrency: number;
+    useCache: boolean;
+    queueMessagesTTL: string;
+    datasetClientCleanerCheckFrequency: string;
+  }
+
+  type StandardCb<T> = (err: Error | string | null, res: T | null) => void;
```

Comment:
Good catch. Null should not be used - undefined is preferred. If you want something to be undefined then `?` should be added. `Error | undefined` => `Error?`